### PR TITLE
License check tools

### PR DESCRIPTION
**Pull request by kirillDanshin**

### CODE REVIEW

**Comment by kirillDanshin** [comment]
Currently github don't recognized the license, so to fix this, in LICENSE should be the actual license, not just a file header

image

You can see a working Apache-2.0 example here https://github.com/audioid/audioid

image

**Comment by flimzy** [comment]
Thanks for raising the issue. I'll examine this more closely soon. I'll need to apply the same change to the other related kivik repos, too.

File: script/license.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2017 Jonathan Hall
```
Comment (kirillDanshin) [inline script/license.txt]: @flimzy can you also consider adding "Copyright (c) [yyyy] Kivik contributors"?

Comment (flimzy) [reply script/license.txt]: I had the same thought earlier today. I'll be doing a massive licensing update.

**Comment by flimzy** [comment]
Resolved in #503

**Pull request closed without merging**